src/util/scroll.js

```diff
@@ -22,7 +23,8 @@ export function setupScroll () {
   // preserve existing history state as it could be overriden by the user
   const stateCopy = extend({}, window.history.state)
   stateCopy.key = getStateKey()
-  window.history.replaceState(stateCopy, '', absolutePath)
+  // Fix for #2593 clean path to avoid crashing entire app on paths with leading double-slash (http://example.com//foo/bar)
+  window.history.replaceState(stateCopy, '', cleanPath(absolutePath))
```

Comment:
Maybe the `absolutePath` should not remove the protocol and path in line 22 but we need to make sure this change works well with `file://` protocol as well

Reply:
I think that would be ideal, but I don't 100% understand what this code is written to accomplish. there aren't any UTs on this method and I wasn't able to get the e2e tests running locally due to a chromedriver/chrome version mismatch I couldn't resolve, so I kinda need to be sure of what I'm doing in order to prevent needing to push random changes to run circleci builds